source/codegen/templates/service_helpers.mako

```diff
@@ -488,11 +512,23 @@ ${initialize_input_param(function_name, param, parameters)}\
 % endfor
 </%def>
 
-<%def name="initialize_service_output_params(output_params)">
+<%def name="initialize_output_streaming_parameters(output_params_to_define)">\
+% for parameter in output_params_to_define:
+<%
+  parameter_name = common_helpers.get_cpp_local_name(parameter)
+%>
+    ${parameter['type']} ${parameter_name} = {};\
+% endfor
+</%def>
+
+<%def name="initialize_service_output_params(output_params, streaming_param)">\
```

Comment:
the name initialize_output_streaming_parameters and initialize_service_output_params, looks same, we should update it accordingly to its usage.

Reply:
Since the purpose "initialize_output_streaming_parameters " is to initialize local variables for output streaming parameters, I believe a better name could be: "initialize_output_streaming_local_variables" or "initialize_streaming_output_variables"
@doshirohan

Reply:
renamed as discussed offline